presto-main/src/main/java/com/facebook/presto/operator/aggregation/FloatSumAggregation.java

```diff
@@ -20,17 +20,18 @@
 import com.facebook.presto.spi.function.InputFunction;
 import com.facebook.presto.spi.function.OutputFunction;
 import com.facebook.presto.spi.function.SqlType;
-import com.facebook.presto.spi.type.FloatType;
 import com.facebook.presto.spi.type.StandardTypes;
+import com.google.common.collect.ImmutableList;
 
+import static com.facebook.presto.spi.type.FloatType.FLOAT;
 import static com.facebook.presto.testing.AggregationTestUtils.generateInternalAggregationFunction;
 import static java.lang.Float.floatToRawIntBits;
 import static java.lang.Float.intBitsToFloat;
 
 @AggregationFunction("sum")
 public final class FloatSumAggregation
 {
-    public static final InternalAggregationFunction FLOAT_SUM = generateInternalAggregationFunction(FloatSumAggregation.class);
+    public static final InternalAggregationFunction FLOAT_SUM = generateInternalAggregationFunction(FloatSumAggregation.class, FLOAT.getTypeSignature(), ImmutableList.of(FLOAT.getTypeSignature()));
```

Comment:
should production code use from `com.facebook.presto.testing.AggregationTestUtils`. It seems that it is test specific.